<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PowerCurveDialog</class>
 <widget class="QDialog" name="PowerCurveDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1048</width>
    <height>853</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Pover Curve Name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="curveNameEdit">
           <property name="maxLength">
            <number>8</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="1" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Time:</string>
           </property>
          </widget>
         </item>
         <item row="8" column="2">
          <widget class="QLabel" name="label_23">
           <property name="text">
            <string>Pct:</string>
           </property>
          </widget>
         </item>
         <item row="6" column="2">
          <widget class="QLabel" name="label_21">
           <property name="text">
            <string>Pct:</string>
           </property>
          </widget>
         </item>
         <item row="7" column="3">
          <widget class="QSpinBox" name="p07Spin">
           <property name="maximum">
            <number>250</number>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Time:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>Pct:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Time:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDoubleSpinBox" name="t02Spin">
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="maximum">
            <double>15.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
         <item row="10" column="1">
          <widget class="QDoubleSpinBox" name="t10Spin">
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="maximum">
            <double>15.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
         <item row="10" column="3">
          <widget class="QSpinBox" name="p10Spin">
           <property name="maximum">
            <number>250</number>
           </property>
          </widget>
         </item>
         <item row="9" column="2">
          <widget class="QLabel" name="label_24">
           <property name="text">
            <string>Pct:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Time:</string>
           </property>
          </widget>
         </item>
         <item row="11" column="1">
          <widget class="QDoubleSpinBox" name="t11Spin">
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="maximum">
            <double>15.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Pct:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>Pct:</string>
           </property>
          </widget>
         </item>
         <item row="11" column="2">
          <widget class="QLabel" name="label_26">
           <property name="text">
            <string>Pct:</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QDoubleSpinBox" name="t06Spin">
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="maximum">
            <double>15.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QDoubleSpinBox" name="t04Spin">
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="maximum">
            <double>15.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>Pct:</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Time:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>Pct:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Time:</string>
           </property>
          </widget>
         </item>
         <item row="11" column="3">
          <widget class="QSpinBox" name="p11Spin">
           <property name="maximum">
            <number>250</number>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Time:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="3">
          <widget class="QSpinBox" name="p05Spin">
           <property name="maximum">
            <number>250</number>
           </property>
          </widget>
         </item>
         <item row="10" column="2">
          <widget class="QLabel" name="label_25">
           <property name="text">
            <string>Pct:</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Time:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>Pct:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QDoubleSpinBox" name="t05Spin">
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="maximum">
            <double>15.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Time:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QSpinBox" name="p00Spin">
           <property name="maximum">
            <number>250</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="t01Spin">
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="maximum">
            <double>15.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QSpinBox" name="p02Spin">
           <property name="maximum">
            <number>250</number>
           </property>
          </widget>
         </item>
         <item row="6" column="3">
          <widget class="QSpinBox" name="p06Spin">
           <property name="maximum">
            <number>250</number>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QSpinBox" name="p01Spin">
           <property name="maximum">
            <number>250</number>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Time:</string>
           </property>
          </widget>
         </item>
         <item row="7" column="2">
          <widget class="QLabel" name="label_22">
           <property name="text">
            <string>Pct:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="t00Spin">
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="maximum">
            <double>15.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
         <item row="11" column="0">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Time:</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QDoubleSpinBox" name="t07Spin">
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="maximum">
            <double>15.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Time:</string>
           </property>
          </widget>
         </item>
         <item row="8" column="3">
          <widget class="QSpinBox" name="p08Spin">
           <property name="maximum">
            <number>250</number>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QSpinBox" name="p03Spin">
           <property name="maximum">
            <number>250</number>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QDoubleSpinBox" name="t03Spin">
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="maximum">
            <double>15.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
         <item row="9" column="3">
          <widget class="QSpinBox" name="p09Spin">
           <property name="maximum">
            <number>250</number>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QDoubleSpinBox" name="t09Spin">
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="maximum">
            <double>15.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
         <item row="4" column="3">
          <widget class="QSpinBox" name="p04Spin">
           <property name="maximum">
            <number>250</number>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QDoubleSpinBox" name="t08Spin">
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="maximum">
            <double>15.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Left-click and drag (&amp;quot;rubber band&amp;quot;) &lt;/p&gt;&lt;p&gt;on chart to zoom in.&lt;/p&gt;&lt;p&gt;Right-click on chart to zoom out&lt;/p&gt;&lt;p&gt;Grab and drag points.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="curveLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <widget class="QPushButton" name="exportBtn">
       <property name="text">
        <string>Export</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="importBtn">
       <property name="text">
        <string>Import</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saveBtn">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBtn">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
